clustering/common/src/main/java/org/jboss/as/clustering/controller/CredentialSourceDependency.java

```diff
@@ -203,6 +204,11 @@ public CredentialSource getValue() {
             throw new IllegalStateException();
         }
 
+        @Override
```

Comment:
Just skipping this @Override lets us skip the coordinated merge.

Don't worry about this for this one; as I'll do the core release now and this coordination is no big deal. It's just something that's happened to me a couple times (oddly enough) and temporarily skipping the @Override was an easy way to not have to juggle.

Reply:
The type LifecycleListener is only in MSC 1.3.0.Final so that would have failed as well in this case.

Reply:
Doh!